packages/frontend/src/pages/CreateService.spec.ts

```diff
@@ -288,6 +288,7 @@ test('should display connectionInfo message if there is no running connection',
       id: 'id',
       file: {
         file: 'file',
+        // eslint-disable-next-line sonarjs/publicly-writable-directories
```

Comment:
does it work using `path.resolve(os.tmpdir(), 'path')` instead ?

Reply:
Done